--- CMakeLists.txt	2016-05-13 18:17:37.000000000 -0300
+++ CMakeLists.txt	2016-06-20 09:51:58.276466121 -0300
@@ -482,9 +482,6 @@
 IF (UNIX AND ${CMAKE_SIZEOF_VOID_P} MATCHES "8")
   ADD_DEFINITIONS(-DLONGLONG_IS_LONG)
   # for easier debugging of cfortran.h
-  IF (APPLE)
-    SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fmacro-backtrace-limit=0")
-  endif ()
 ENDIF()
 
 # Determine C/Fortran pointer compatibility.
